### ISSUE TITLE
Results of training on my own dataset

### ISSUE THREAD

**Issue by dongdongdashen** [CorentinJ/Real-Time-Voice-Cloning#1029]
Hi,When I train with my own dataset, the result is as shown in the following picture.Can you tell me where is the problem? May be the number of data is too small? (my dataset includes 20 minutes of voice data in total ) Looking forward to your reply :
![step-5000-align](https://user-images.githubusercontent.com/40753397/155947128-d47f01a3-e1b8-4ee7-899f-fa35ec323a5a.png)
![step-5000-mel-spectrogram](https://user-images.githubusercontent.com/40753397/155947145-435597eb-fc87-4c4f-b357-fc461cd87e10.png)
)

**Comment by ireneb612** [comment]
Yes 20 minutes is nt enough! you can use this much data if you fine tune on pre trained weights!
